### PR TITLE
implemented user authentication with Google OAuth2

### PR DESCRIPTION
**Pull request by Timothy-py**
Implementation with googleapis and without Passport

### CODE REVIEW

**Comment by Timothy-py** [comment]
@romzy234

**Comment by Timothy-py** [comment]
@romzy234

File: model/README.md [hunk 1]
```diff
@@ -1,236 +1,173 @@
...
-| Phone | String/Number - Require - True |
-| About     | String - Require - False|
-| Socials | Array[Object] |
-| Verify | Boolean - Require - True |
-| Hide_Detail | Boolean - Require - True |
-
-### 4. Sponsor
-
-This Table Manages The Sponsor Post  
+### 2. Google Auth
```
Comment (romzy234) [inline model/README.md]: This table isn't needed

**Review by romzy234 — APPROVED** [review]
Let Wrap this up

**Pull request merged**